ipam.go

```diff
@@ -182,3 +182,30 @@ func Free(network net.IPNet, mask net.IPMask, subnets []net.IPNet) (net.IPNet, e
 
 	return freeNetwork, nil
 }
+
+// Half takes a network and returns two subnets which split the network in
```

Comment:
`split` -> `splits`

Reply:
`split` is correct. The two networks split the larger network.